web-app/client/src/components/LogInModal/steps/Code.tsx

```diff
@@ -0,0 +1,95 @@
+import { useMutation } from '@apollo/client';
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import Button from '@components/Button';
+import { Text } from '@components/Inputs';
+import {
+  approveRecoveryCode,
+  approveRecoveryCodeVariables,
+} from '@graphql/operations/mutations/__generated__/approveRecoveryCode';
+import {
+  issueCodeForPasswordRecovery,
+  issueCodeForPasswordRecoveryVariables,
+} from '@graphql/operations/mutations/__generated__/issueCodeForPasswordRecovery';
+import { APPROVE_RECOVERY_CODE } from '@graphql/operations/mutations/approveRecoveryCode';
+import { ISSUE_CODE_FOR_PASSWORD_RECOVERY } from '@graphql/operations/mutations/issueCodeForPasswordRecovery';
+import styles from '../LogInModal.module.scss';
+
+type Inputs = {
+  code: string;
+};
+
+const defaultValues: Inputs = {
+  code: '',
+};
+
+interface Props {
+  email: string;
+  onSuccess: () => void;
+}
+
+const Code: React.FC<Props> = ({ onSuccess, email }) => {
```

Comment:
```suggestion
const Code: FC<Props> = ({ onSuccess, email }) => {
```
Надо придерживаться единого стиля ;)